builder/win/NSIS_Installer.nsi

```diff
@@ -226,6 +233,10 @@ Section $(MsgAssoc) assoc
 SectionEnd ; end of file association section
 
 Section /o $(MsgRunAtStart) startup
+  ;------------------------------------------------------------------
+  ; Makes sure shell folders (e.g. $SMPROGRAMS and $DESKTOP) use the "all users" location
+    SetShellVarContext all
```

Comment:
From the documentation I read that it needs to be set seperatly for installer and Uninstaller, so is it really needed 4x?

Reply:
Most likely not, but I erred on the side of caution. I've reviewed the occurrences and eliminated some.